src/components/latestPosts/latestPosts.tsx

```diff
@@ -0,0 +1,36 @@
+import clsx from "clsx";
+import React from "react";
+import styles from "./styles.module.css";
+
+export default function LatestPosts({ recentPosts }) {
+  return (
+    <div className={clsx("container", styles.wrap)}>
```

Comment:
Would it be worthwhile using Infima's card component for the blog posts over a list.